sql/core/src/test/resources/sql-tests/results/pgSQL/with.sql.out

```diff
@@ -358,23 +358,20 @@ create table foo (with baz)
 
 
 -- !query 38
--- fail, WITH is a reserved word
 create table foo (with ordinality)
 -- !query 38 schema
 struct<>
 -- !query 38 output
 org.apache.spark.sql.catalyst.parser.ParseException
 
-no viable alternative at input 'with'(line 2, pos 18)
+no viable alternative at input 'with'(line 1, pos 18)
 
 == SQL ==
--- fail, WITH is a reserved word
 create table foo (with ordinality)
 ------------------^^^
 
 
 -- !query 39
--- fail, WITH is a reserved word
```

Comment:
Actually, these comments were helpful for me to review your `PostgreSQL` PRs to compare the result.

Reply:
At least we need to remove these comment because it is incorrect:
https://github.com/apache/spark/pull/25357#discussion_r310429369